tests/run-make/short-ice/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+extern crate error_lib;
```

Comment:
As said before, could you prefer a type error, that would better show what you are trying to do.

Reply:
yes, for type errors we could remove several parts of backtraces from query system:

[pre-short.log](https://github.com/rust-lang/rust/files/10951339/pre-short.log)

[now-short.log](https://github.com/rust-lang/rust/files/10951340/now-short.log)

FYI @jyn514